Author: Vytautas Dauksa <vytas.dauksa@smoothwall.net>
Description: fixed usr/bin/{bzcmp,bzegrep,bzfgrep,bzless} linking.
 It were linking to non existing locations. This fixes them.
--- a/Makefile
+++ b/Makefile
@@ -90,14 +90,14 @@
 	cp -f libbz2.a $(PREFIX)/lib
 	chmod a+r $(PREFIX)/lib/libbz2.a
 	cp -f bzgrep $(PREFIX)/bin/bzgrep
-	ln -s -f $(PREFIX)/bin/bzgrep $(PREFIX)/bin/bzegrep
-	ln -s -f $(PREFIX)/bin/bzgrep $(PREFIX)/bin/bzfgrep
+	(cd $(PREFIX)/bin; ln -s -f bzgrep bzegrep)
+	(cd $(PREFIX)/bin; ln -s -f bzgrep bzfgrep)
 	chmod a+x $(PREFIX)/bin/bzgrep
 	cp -f bzmore $(PREFIX)/bin/bzmore
-	ln -s -f $(PREFIX)/bin/bzmore $(PREFIX)/bin/bzless
+	(cd $(PREFIX)/bin; ln -s -f bzmore bzless)
 	chmod a+x $(PREFIX)/bin/bzmore
 	cp -f bzdiff $(PREFIX)/bin/bzdiff
-	ln -s -f $(PREFIX)/bin/bzdiff $(PREFIX)/bin/bzcmp
+	(cd $(PREFIX)/bin; ln -s -f bzdiff bzcmp)
 	chmod a+x $(PREFIX)/bin/bzdiff
 	cp -f bzgrep.1 bzmore.1 bzdiff.1 $(PREFIX)/man/man1
 	chmod a+r $(PREFIX)/man/man1/bzgrep.1
